certified-connectors/Advanced Data Operations/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1389 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Advanced Data Operations",
+    "version": "1.0.0",
+    "description": "A suite of advanced API's designed to assist in the transformation of bulk datasets.",
```

Comment:
@brad302 please remove the word `API` or replace it with other word (e.g. services) in the description,

Reply:
Thanks @v-EgorKozhadei, that's done.